account_payment_order/migrations/16.0.1.0.0/post-migration.py

```diff
@@ -161,6 +164,18 @@ def _insert_payment_line_payment_link(env):
 
 @openupgrade.migrate()
 def migrate(env, version):
+    env.cr.execute(
```

Comment:
Do we want to execute this when this has already been executed during `14.0.2.0.0` migration?
This looks valid only if migrating from `14.0.1.0.0` to `16.0.1.0.0`.
Same applies to `14.0.3.0.0`